public/views/partials/lib/commands/user/create/check.liquid

```diff
@@ -0,0 +1,19 @@
+{% liquid
+  assign c = '{ "errors": {}, "valid": true }' | parse_json
+
+  function c = 'modules/core/lib/validations/presence', c: c, object: object, field_name: 'email', key: 'modules/user/validation.email.required'
+  function c = 'modules/core/lib/validations/email', c: c, object: object, field_name: 'email', key: 'modules/user/validation.email.format'
+
+  if object.email != blank
+    graphql user_exists = 'modules/user/user/exists', email: object.email
```

Comment:
I think it would be better to extend the module's `load` query to enable searching by email too. In this case we won't need a new query. What do you think?